### PR TITLE
Add status badge for main plawright run

### PR DESCRIPTION
**Pull request by kornys**

### CODE REVIEW

**Comment by kornys** [comment]
@rhosak-ci-bot test

**Comment by Frawless** [comment]
Test

**Review by Frawless — COMMENTED** [review]
diff test

File: Readme.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # E2E tests for console.redhat.com
+[![Playwright Tests](https://github.com/redhat-developer/consoledot-e2e/actions/workflows/playwright.yml/badge.svg?branch=main)](https://github.com/redhat-developer/consoledot-e2e/actions/workflows/playwright.yml)
```
Comment (Frawless) [inline Readme.md:2]: NO!

Comment (andreaTP) [reply Readme.md:2]: would you mind elaborating?

Comment (kornys) [reply Readme.md:2]: @andreaTP all these comments are just for testing of trigger jenins via github PR comment

**Comment by kornys** [comment]
@rhosak-ci-bot test

**Comment by kornys** [comment]
@rhosak-ci-bot test

**Review by Frawless — APPROVED** [review]

**Pull request merged**